### PR TITLE
Heroku deployment README

### PR DESCRIPTION
**Pull request by rwz**
Had to add Heroku deployment guide. Could you guys proofread and review? @rondale-sc @seanpdoyle

### CODE REVIEW

**Comment by rwz** [comment]
It's heavily based on @seanpdoyle README from https://github.com/seanpdoyle/ember-cli-rails-heroku-example BTW.

File: README.md [hunk 1]
```diff
@@ -132,6 +132,57 @@ This tells Guard to watch your EmberCLI app for any changes to the JavaScript,
...
+
+Add `rails_12factor` gem to your production group in Gemfile, then run `bundle
+install`:
+
+```ruby
+gem "rails_12factor", group: :production
+```
+
+Add a `postinstall` task to your Ember CLI app's `package.json`. This will
+ensure that during the deployment process, Heroku will all dependencies found
```
Comment (seanpdoyle) [inline README.md]: > Heroku will all dependencies found

"Heroku will install all dependencies found"

File: README.md [hunk 2]
```diff
@@ -132,6 +132,57 @@ This tells Guard to watch your EmberCLI app for any changes to the JavaScript,
...
+Add a `package.json` to the root of your Rails project. This is to make sure
+it'll be detected by the NodeJS buildpack.
+
+```javascript
+{
+  "dependencies": {
+    "bower": "*"
+  }
+}
+```
```
Comment (seanpdoyle) [inline README.md]: How do you feel about putting this before the `"scripts"` code snippet?

Comment (rwz) [inline README.md]: Good point

**Pull request merged**